tests/nodeunit/unit/kujua-sms/updates.js

```diff
@@ -462,27 +462,28 @@ exports['creates record with empty message field'] = function(test) {
     test.done();
 };
 
-exports['JSON POST: return 500 error if form not found'] = function(test) {
+exports['JSON POST: return 404 error if form not found'] = function(test) {
     var req = {
-        body: '{ "_meta: { "form: "foo" } }'
```

Comment:
Yeah. It didn't work. The test passed because is wasn't checking the error thoroughly enough. So when the parsing failed with a cannot parse JSON error the test passed.